Compilation fails

If I start compiling 5b580fe8dfcdb63ba544c2d50fbbad15b4e6d858, then the following error is raised:

```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project uv-e-skGovCRP: Compilation failure: Compilation failure:
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skGovCRP/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skgovcrp/SkGovCRP.java:[12,32] package org.apache.commons.lang3 does not exist
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skGovCRP/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skgovcrp/SkGovCRP.java:[219,30] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skgovcrp.SkGovCRP
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skGovCRP/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skgovcrp/SkGovCRP.java:[291,25] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skgovcrp.SkGovCRP
[ERROR] /Users/mynarzjindrich/Dev/UnifiedViews/comsode-uv-plugins/e-skGovCRP/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skgovcrp/SkGovCRP.java:[292,18] cannot find symbol
[ERROR] symbol:   variable StringUtils
[ERROR] location: class eu.comsode.unifiedviews.plugins.extractor.skgovcrp.SkGovCRP
```

--- a/pom.xml
+++ b/pom.xml
@@ -36,19 +36,20 @@
 		<dependency>
 			<groupId>commons-io</groupId>
 			<artifactId>commons-io</artifactId>
+			<scope>compile</scope>
 		</dependency>
 		<dependency>
 			<groupId>org.apache.httpcomponents</groupId>
 			<artifactId>httpclient-osgi</artifactId>
 			<version>4.3.3</version>
-			<scope>provided</scope>
+			<scope>compile</scope>
 		</dependency>
 		<dependency>
-      		<!-- jsoup HTML parser library @ http://jsoup.org/ -->
-      		<groupId>org.jsoup</groupId>
-      		<artifactId>jsoup</artifactId>
-      		<version>1.8.2</version>
-    	</dependency>
+			<!-- jsoup HTML parser library @ http://jsoup.org/ -->
+			<groupId>org.jsoup</groupId>
+			<artifactId>jsoup</artifactId>
+			<version>1.8.2</version>
+		</dependency>
 		<!-- UNIT Test dependencies. -->
 		<dependency>
 			<groupId>junit</groupId>
@@ -60,6 +61,11 @@
 			<groupId>eu.unifiedviews</groupId>
 			<artifactId>module-test</artifactId>
 			<scope>test</scope>
+		</dependency>
+		<dependency>
+			<groupId>org.apache.commons</groupId>
+			<artifactId>commons-lang3</artifactId>
+			<version>3.1</version>
 		</dependency>
 	</dependencies>
 

--- a/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skgovcrp/SkGovCRPVaadinDialog.java
+++ b/src/main/java/eu/comsode/unifiedviews/plugins/extractor/skgovcrp/SkGovCRPVaadinDialog.java
@@ -8,6 +8,8 @@ import eu.unifiedviews.helpers.dpu.vaadin.dialog.AbstractDialog;
  */
 public class SkGovCRPVaadinDialog extends AbstractDialog<SkGovCRPConfig_V1> {
 
+    private static final long serialVersionUID = 8715849977476398760L;
+
     public SkGovCRPVaadinDialog() {
         super(SkGovCRP.class);
     }

